src/providers/data_provider_be.c

```diff
@@ -615,6 +620,59 @@ errno_t be_process_init(TALLOC_CTX *mem_ctx,
     return ret;
 }
 
+static void watch_update_resolv(const char *filename, void *arg)
+{
+    int ret;
+
+    DEBUG(SSSDBG_TRACE_FUNC, "Reloading %s.\n", filename);
+
+    ret = res_init();
+    if (ret != 0) {
+        DEBUG(SSSDBG_OP_FAILURE, "Failed to reload %s.\n", filename);
+        return;
+    }
+}
+
+static int watch_config_files(struct be_ctx *ctx)
+{
+    int ret;
+    bool monitor_resolv_conf;
+    bool use_inotify;
+
+    /* Watch for changes to the DNS resolv.conf */
+    ret = confdb_get_bool(ctx->cdb,
+                          CONFDB_MONITOR_CONF_ENTRY,
+                          CONFDB_MONITOR_RESOLV_CONF,
+                          true, &monitor_resolv_conf);
+    if (ret != EOK) {
+        return ret;
```

Comment:
DEBUG(... error message here?

Reply:
`conf_get_bool()` already logs a debug message in case of error.